docs/output.md

```diff
@@ -1,40 +1,45 @@
+<!-- omit in toc -->
```

Comment:
I usually put it at the end of the line that has the stuff I want to omit, but it does make sense to put it before as well